src/pykeen/models/base.py

```diff
@@ -50,9 +50,19 @@ class Model(nn.Module, ABC):
     #: The instance of the loss
     loss: Loss
 
+    #: the number of entities
     num_entities: int
+    #: the number of relations
     num_relations: int
+    #: whether to use inverse relations
     use_inverse_triples: bool
+    #: utility for generating inverse relations
+    relation_inverter: RelationInverter
```

Comment:
Is this only set in the new-style base model?

Reply:
yes, this is used in the highest level model class